samples/generated/express-direct-auth/config.js

```diff
@@ -12,9 +12,9 @@ module.exports = function () {
     webServer: {
       port: 8080,
       oidc: {
-        clientId: CLIENT_ID || '',
-        clientSecret: CLIENT_SECRET || '',
-        issuer: ISSUER || '',
+        clientId: CLIENT_ID,
```

Comment:
are these changes intentional?

Reply:
I think it's changed by Vijet, this PR just generates the new samples based on the template file.